src/OneExplorer/OneExplorer.ts

```diff
@@ -1026,8 +1028,56 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
   }
 
   /**
-   * Create ONE configuration file for a base model
-   * Input box is prefilled as <base model's name>.cfg
+   * Select the option for the configuration you want to create
+   * Return information about the selected option
+   *
+   * @param modelName A base model's name
+   * @param extName A base model's extension name
+   *
+   */
+  private async generateCfgInfo(
+    modelName: string,
+    extName: string
+  ): Promise<CfgInfo | undefined> {
+    const options: vscode.QuickPickItem[] = [
+      { label: ".cfg", description: "configuration file of onecc compiler" },
+    ];
+
+    if (extName === "tflite") {
```

Comment:
Thanks that you change this to show two options only if its' tflite.
However, the other cases (*.onnx, *.pb) still shows the quickpick with "ONLY ONE CHOICE". It's meaningless. could you make it skip the quickpick selection if not tflite?

Reply:
Because I think cfg files are likely to be created with other extensions in the future, instead of skipping `.tflite` files, I refactor them so that they can be skipped if there is only one option to choose from.